src/pages/students/[student_id]/goals/[goal_id]/create.tsx

```diff
@@ -27,11 +28,21 @@ interface BenchmarkFormEntry {
   [key: string]: string | number | "";
 }
 
+const BenchmarkStepperIcon = (stepIconProps: StepIconProps) => {
+  const { completed = false } = stepIconProps;
+
+  if (completed) {
+    return <CheckCircle />;
+  } else {
+    return <TripOriginRounded />;
+  }
```

Comment:
Since the Stepper will set `completed` appropriately whether or not the step for the current icon has been completed, we can keep track of which icon to use here.